### PR TITLE
Kpf version1 todos3

### PR DESCRIPTION
**Pull request by kylepfeeley**

### CODE REVIEW

File: src/routes/form/Wizard.js [hunk 1]
```diff
@@ -108,17 +92,12 @@ class Wizard extends Component {
...
     handleSubmit = (values) => {
-      // getting aff values
       values.client_name= this.props.affid || 'HealthDefault';
       values.sub_id1= this.props.sub_id1 || 's1';
       values.sub_id2= this.props.sub_id2 || 's2';
       values.sub_id3= this.props.sub_id3 || 's3';
 
-      // hardcoding some values
-      // values.user_agent= 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3776.0 Safari/537.36';
       values.ip_address = '127.0.0.1';
```
Comment (guyandy) [inline src/routes/form/Wizard.js]: Does this get overwritten later?

Comment (kylepfeeley) [reply src/routes/form/Wizard.js]: @ukozazenje @perichin I think you guys added these. Can you comment?

**Comment by kylepfeeley** [comment]
@guyandy Closing this and opening it for `dev` branch instead.

File: src/routes/form/Wizard.js (hunk 1 above)
Comment (ukozazenje) [reply src/routes/form/Wizard.js]: Yes, That is in case we don't get the ip_address from API

**Pull request closed without merging**